Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2457,6 +2458,11 @@ - (void)handleInfoMessageTap:(TSInfoMessage *)message
             OWSAssert([message isKindOfClass:[OWSAddToContactsOfferMessage class]]);
             [self tappedAddToContactsOfferMessage:(OWSAddToContactsOfferMessage *)message];
             return;
+        case TSInfoMessageAddUserToProfileWhitelistOffer:
+        case TSInfoMessageAddGroupToProfileWhitelistOffer:
```

Comment:
This PR adds a "add to profile whitelist" offer.  There are slightly different flavors for users and groups, since they (may) have different copy.